vault/identity_store_oidc_provider.go

```diff
@@ -63,6 +74,50 @@ func oidcProviderPaths(i *IdentityStore) []*framework.Path {
 			HelpSynopsis:    "List OIDC assignments",
 			HelpDescription: "List all configured OIDC assignments in the identity backend.",
 		},
+		{
+			Pattern: "oidc/scope/" + framework.GenericNameRegex("name"),
+			Fields: map[string]*framework.FieldSchema{
+				"name": {
+					Type:        framework.TypeString,
+					Description: "Name of the scope",
+				},
+				"template": {
+					Type:        framework.TypeString,
+					Description: "The template string for the scope",
```

Comment:
nit: We could improve on the description here a bit. See the description for [identity_store_oidc.go#L249](https://github.com/hashicorp/vault/blob/main/vault/identity_store_oidc.go#L249).

Maybe something like:
> The template string to use for the scope. This may be in string-ified JSON or base64 format.